docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+# Scalar DL Backup Creation and Restoration
+
+Scalar DL backup should be created after the Scalar DL server is paused otherwise there is a chance for inconsistency. 
+This guide shows you how to create and restore Scalar DL data backup without any data inconsistency.
+
+## Prerequisites
+
+* Read the Guide on How to [Back up and Restore Databases Integrated with Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md)
+* Scalar DL must be deployed in the Kubernetes cluster
+* Cosmos DB account must be created with the backup policy `continuous` if you are using Cosmos DB.
+* You must synchronize time on the ledger and auditor servers if you are using the audit service.
```

Comment:
```suggestion
* You must synchronize the clocks (moderately) between Ledger and Auditor servers by using clock synchronization mechanisms such as NTP if you use Auditor. 
```

Reply:
Thank you for your suggestion.
Suggestion integrated cd32752